sdf/1.7/frame.sdf

```diff
@@ -1,16 +1,32 @@
 <!-- Frame -->
 <element name="frame" required="*">
-  <description>A frame of reference to which a pose is relative.</description>
+  <description>A frame of reference in which poses may be expressed.</description>
```

Comment:
can you copy the change on this line to `sdf/1.8/frame.sdf`?

Reply:
@scpeters Sure think ... done :)